lib/features/send/send_confirmation_page.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:didpay/shared/success_page.dart';
+import 'package:didpay/shared/pending_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:didpay/l10n/app_localizations.dart';
+
+// TODO: make HookConsumerWidget after pfi is implemented
+class SendConfirmationPage extends HookWidget {
```

Comment:
Should we add a test for this one too? To ensure it shows the right page based on the state here?